bin/deploy

```diff
@@ -164,11 +168,24 @@ main() {
 
   local conda_packages
   conda_packages="conda${pyver_prefix}_${deploy_mode}-${pkg_postfix}.txt"
-  local env_file="${SCRIPT_DIR}/../etc/${conda_packages}"
-  local lsstsw_ref
-  lsstsw_ref=$(git log -n 1 --pretty=format:'%h' "$env_file")
 
-  echo "::: effective LSST_LSSTSW_REF: ${lsstsw_ref}"
+  cd "$LSSTSW"
+  # conda environment reference
+  local env_file="${LSSTSW}/env/${LSST_SPLENV_REF}/${conda_packages}"
+  local env_url="https://raw.githubusercontent.com/lsst/scipipe_conda_env/${LSST_SPLENV_REF}/etc/"
+
+  echo "::: conda environment file: ${env_file}"
+
+  cd env
+  if [ -d "${LSST_SPLENV_REF}" ]; then
```

Comment:
It would be better to check the complete local file path as otherwise, trying to switch from a "bleed" deploy to regular (or vise versa), the proper conda env file wouldn't be retrieved.